firestore/proguard-rules.pro

```diff
@@ -1 +1,2 @@
 -dontwarn com.firebase.ui.firestore.paging.**
+-dontwarn com.firebase.ui.database.paging.**
```

Comment:
We should not need to mention the database adapter in the firestore package.  I think this is in the wrong file.